include/osquery/config.h

```diff
@@ -22,7 +22,7 @@
 namespace osquery {
 
 /// The builder or invoker may change the default config plugin.
-DECLARE_string(config_retriever);
+DECLARE_string(config_plugin);
```

Comment:
nice. let's be sure to update the docs for this as well as logger. also, this may mean we should make this a `1.4` release instead of a minor dot release.